framework/resources/Functional/functions/split/split.json

```diff
@@ -0,0 +1,22 @@
+{
+  "testId": "coalesce",
```

Comment:
I think coalesce was used? Change to "split"?

Reply:
Fixed.